Payments/Operation/02_ManagePaymentsAndCaptures.md

```diff
@@ -0,0 +1,103 @@
+        Actindo has got the response from the service payment provider, that the transfer was successfull.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the *Status information* field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void payment and capture transaction](#void-payment-and-payment-and-capture-transaction).
+    
+2. Click on a transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
+
+    *Payments > Payments and Captures > Tab LIST > Select a payment and capture transaction*   
+
+    ![Payment and capture attributes and logs](../../Assets/Screenshots/Payments/PaymentsCaptures/CheckAttributes.png "[Payment and capture attributes and logs]")   
+
+3. Check the attributes and logs of the transaction. For detailed information, see the following:
+     - [LIST &ndash; Attributes](../UserInterface/03_ListPaymentsAndCaptures.md#payments-and-captures-–-attributes)
+     - [LIST &ndash; Logs](../UserInterface/03_ListPaymentsAndCaptures.md#payments-and-captures-–-logs)
+4. Return to the LIST and decide how to proceed with the transaction.
+
+
+
+## Refund payment and capture transaction
+
+After you have checked a payment and capture transaction that could not be processed by the workflow, you can refund the payment and capture transaction manually, so that the order process can be cancelled and the customer can get his or her money back.<!---ist das richtig-->
+
+
+#### Prerequisites
+
+The status of a payment and capture transaction is **not** success. In this case, further processing has already been continued. <!---ist das richtig-->
+
+#### Procedure
+
+*Payments > Payments and Captures > Tab LIST*   
+
+![LIST (Payments and captures)](../../Assets/Screenshots/Payments/PaymentsCaptures/ChangePaymentsCaptures.png "[LIST (Payments and captures]")   
+
+1. Check the payment and capture transaction you want to refund, see [Check payment and capture transaction](#check-payment-and-capture-transaction).
+2. Select the payment and capture transaction you want to refund by clicking the checkbox on the left.   
+    The editing toolbar is displayed.
+3. Click the [Refund] button. <!---was passsiert dann-->   
+   A confirmation message is displayed. The status of the payment and capture transaction has changed to *Success*. <!---Stefan, oder "Unconfirmed"?-->   
+   You can now follow up this case under the menu entry *Refunds*. <!-----Stefan ist das richtig?--> 
+
+
+
+## Void payment and capture transaction
+
+You can void a payment and capture transaction, if you want to disable the transaction for the Actindo database. 
+> [Info] The payment service provider will not be informed about changing the status to "Void". It is only done to clear the Actindo database.
```

Comment:
```suggestion
> [Info] The payment service provider will not be informed about changing the status to **Void**. It is only done to clear the Actindo database.
```
"Clear the database": Vielleicht kann das missvestanden werden? Ich vermute, es wird nicht gelöscht von der Datenbank, sondern nur als "Void" markiert, damit die Transaktion nicht "hängend" bleibt? Stefan meinte "for houskeeping purposes".

Reply:
darum clear, dann ist es sauber